### ISSUE TITLE
cssnano doesn't minify when called via esbuild-postcss

### ISSUE THREAD

**Issue by joliss** [karolis-sh/esbuild-postcss#8]
I'm encountering a very strange issue. Perhaps @ludofischer from the cssnano project might have an idea as well.

I'm calling `esbuild` like so:

```js
esbuild.build({
  entryPoints: ['src/app.tsx', 'src/app.css'],
  bundle: true,
  plugins: [postcss()],
  outdir: 'dist',
})
```

And my `postcss.config.cjs` reads as follows:

```js
module.exports = {
  map: false,
  plugins: {
    'postcss-import': {},
    "tailwindcss/nesting": {},
    tailwindcss: {},
    autoprefixer: {},
    cssnano: { preset: 'default', },
  },
}
```

When I run `npx postcss src/app.css` directly, it prints minified CSS.

But when I run the `esbuild.build` function above (`rm -rf dist && node esbuild.config.js && cat dist/app.css`), it doesn't minify.

I tried commenting out the `cssnano: { preset: 'default', }` line to see if it makes any difference, and it turns out that cssnano *does* do some things, like combine `border-width` and `border-color` into a single `border` property. It just doesn't minify.

If you want to try and reproduce it, my code for this is at https://github.com/joliss/calculator.

I'm very confused as to why this might be happening.

**Comment by ludofischer** [comment]
cssnano just runs a series of PostCSS plugins one after another so if some transformations happen and not other, it means only some PostCSS plugins run and not others (there is no single 'minification' step). As everything works as expected with the `postcss` command it might be `esbuild-postcss` is preventing cssnano from loading some PostCSS plugins, but I can't say much else as I am not familiar with esbuild plugins or how esbuild-postcss works.

**Comment by karolis-sh** [comment]
Tested a bit, and it seems the minification is actually been done by nano plugins at https://github.com/karolis-sh/esbuild-postcss/blob/main/src/index.ts#L28. It's just that esbuild further analyses that CSS and restores all the newlines and stuff.
But esbuild can handle the actual file content minification itself - https://esbuild.github.io/api/#minify. I added a `minify: true` in your esbuild config, and it worked as expected.
